app/jobs/iiif_print/create_relationships_job_decorator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# OVERRIDE IIIF Print v1.0.0 to call SetDefaultParentThumbnailJob
+
+module IiifPrint
+  module Jobs
+    module CreateRelationshipsJobDecorator
+      def add_to_work(child_record:, parent_record:)
+        return true if parent_record.ordered_members.to_a.include?(child_record)
+
+        set_default_parent_thumbnail(child_record, parent_record) if child_record&.intermediate_file?
+
+        parent_record.ordered_members << child_record
+        @parent_record_members_added = true
```

Comment:
What is the purpose of setting this instance variable and how does it get used?

Reply:
Hmm, this was actually from here https://github.com/scientist-softserv/iiif_print/blob/main/lib/iiif_print/jobs/create_relationships_job.rb#L114

I wasn't sure if I could remove it or not.